app/views/case_study/show.html.erb

```diff
@@ -0,0 +1,71 @@
+<% content_for :title, "#{@case_study_presenter.page_title} - #{I18n.t("formats.#{content_item.document_type}", count: 1)} - GOV.UK" %>
+
+<% content_for :extra_headers do %>
+  <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: content_item.to_h, schema: :article } %>
+<% end %>
+
+<main role="main" id="content" class="case-study" <%= lang_attribute(content_item.locale) %>>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds responsive-top-margin">
+      <%= render 'govuk_publishing_components/components/title', {
+        title: content_item.title,
+        context: I18n.t("formats.#{content_item.document_type}", count: 1),
+        context_locale: t_locale_fallback("formats.#{content_item.document_type}", count: 1),
+        average_title_length: "long",
+      } %>
+    </div>
+    <%= render 'shared/translations' %>
+    <div class="govuk-grid-column-two-thirds">
+      <%= render 'govuk_publishing_components/components/lead_paragraph', text: content_item.description %>
+    </div>
+  </div>
+
+  <%= render 'shared/publisher_metadata', locals: {
+    from: govuk_styled_links_list(content_item.related_entities),
+    first_published: display_date(content_item.first_public_at || content_item.first_published_at),
+    last_updated: display_date(content_item.updated),
+    see_updates_link: true,
+    }
+  %>
+
+  <% if content_item.withdrawn? %>
+    <% withdrawn_time_tag = tag.time(display_date(content_item.withdrawn_at), datetime: content_item.withdrawn_at) %>
+
+    <%= render 'govuk_publishing_components/components/notice', {
+      title: I18n.t("formats.case_study.withdrawn_notice.title", schema_name: I18n.t("formats.#{content_item.schema_name}", count: 1, locale: :en).downcase, withdrawn_time: withdrawn_time_tag).html_safe,
+      description_govspeak: content_item.withdrawn_explanation&.html_safe,
+      time: withdrawn_time_tag,
+      lang: "en",
+    }  %>
+  <% end %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="content-bottom-margin">
```

Comment:
I think we need to copy over the [content-bottom-margin helper stylesheet](https://github.com/alphagov/government-frontend/blob/main/app/assets/stylesheets/helpers/_content-bottom-margin.scss) and import in [application.scss](https://github.com/alphagov/government-frontend/blob/main/app/assets/stylesheets/application.scss#L17) which will add the missing spacing here (I just spotted this for call for evidence docs and also in your screenshots)